data/alpha/register/registration-district.yaml

```diff
@@ -7,4 +7,4 @@ fields:
 phase: alpha
 register: registration-district
 registry: hm-passport-office
-text: Registration districts in England and Wales, defined for the purposes of the Registration Acts
+text: Registration districts in England and Wales, defined for the purposes of the Civil Registrations Acts
```

Comment:
@endamccormack I was too late, it had already been merged, along with a relevant comment below.